### PR TITLE
[10.8] [PR 4500] Move branch specific attributes from global site.yml to branch relevant antora.yml

### PR DESCRIPTION
**Pull request by mmattel**
Backport of PR #4500

### CODE REVIEW

File: antora.yml [hunk 1]
```diff
@@ -8,3 +8,9 @@ nav:
 
 asciidoc:
   attributes:
+    minimum-php-version: 7.3
```
Comment (phil-davis) [inline antora.yml:11]: 10.8 supports PHP 7.2 - I guess you will do a separate PR to update the specific values?

**Review by phil-davis — APPROVED** [review]

File: antora.yml (hunk 1 above)
Comment (mmattel) [reply antora.yml:11]: yes, this PR should just move the attribues as they are.
the next PR is 10.8 (resp 10.7 only) and contains the correct values
